tests/test_metrics.py

```diff
@@ -191,7 +191,10 @@ def test_NegativeBinomialDistributionLoss(center, transformation):
         )
         samples = loss.sample(rescaled_parameters, 1)
         assert torch.isclose(target.mean(), samples.mean(), atol=0.1, rtol=0.5)
-        assert torch.isclose(target.std(), samples.std(), atol=0.1, rtol=0.5)
+        if transformation == "log1p" and not center:
+            assert torch.isclose(target.std(), samples.std(), atol=0.1, rtol=0.8)
```

Comment:
The variance distorts when the data is not centered, as the `log1p` transformation is `log(1+x)`